Server/Server/Metrics/Player.cs

```diff
@@ -0,0 +1,33 @@
+using Server.Settings.Structures;
+
+namespace Server.Metrics {
+    public class Player {
+        public static readonly Prometheus.Gauge Total = Prometheus.Metrics.CreateGauge(
+            "lmp_player_online_total",
+            "The total count of unique players online.",
+            new Prometheus.GaugeConfiguration{}
+        );
+
+        public static readonly Prometheus.Counter Online = Prometheus.Metrics.CreateCounter(
```

Comment:
As far as I remember the Prometheus recommendations, this should probably also be a gauge as it isn't increasing continuously.

Reply:
Yeah, that's correct. This should definitely be a gauge, not sure why I created it as a counter.